### PR TITLE
make it possible to call precompute seperately

### PR DESCRIPTION
**Pull request by himanshuchawla009**
- Allow clients to call precompute separately for ecdsa signing. This can help clients to optimizing signing flow

### CODE REVIEW

File: demo/redirect-flow-example/src/App.tsx [hunk 1]
```diff
@@ -398,7 +382,36 @@ function App() {
...
+    }
+  };
+
+  const signMultipleMessagesWithPrecomputedTss = async (): Promise<any> => {
+    if (coreKitInstance.keyType === "secp256k1") {
+      const precomputedTssClient = await coreKitInstance.precompute_secp256k1();
+      const precomputedTssClient2 = await coreKitInstance.precompute_secp256k1();
+
+      const msg = Buffer.from("hello signer!");
+      const sig = await coreKitInstance.sign(msg, false, precomputedTssClient);
```
Comment (ieow) [inline demo/redirect-flow-example/src/App.tsx:404]: should use Promis.all to showcase parallel for precompute and sign

**Review by matthiasgeihs — COMMENTED** [review]
lgtm

File: demo/redirect-flow-example/src/App.tsx (hunk 1 above)
Comment (himanshuchawla009) [reply demo/redirect-flow-example/src/App.tsx:403]: done

**Review by ieow — APPROVED** [review]
lgtm

File: demo/redirect-flow-example/src/App.tsx (hunk 1 above)
Comment (metalurgical) [reply demo/redirect-flow-example/src/App.tsx:403]: `Promise.all()` will not make it parallel. There is distinct difference between concurrent and parallel computation.

**Pull request merged**